src/main/resources/META-INF/sql/mysql-schema.sql

```diff
@@ -122,4 +122,17 @@ CREATE TABLE IF NOT EXISTS tokens (
   description varchar(128),
   token varchar(1024),
   UNIQUE (role)
+)ENGINE=InnoDB CHARACTER SET utf8;
```

Comment:
I think we should drop mysql stuff
We are not supporting it any more

Reply:
I don't think this schema of MySQL should be deleted, because among the users I come into contact with, too many users are using MySQL database, and they have been complaining about why pulsar-manager cannot conveniently use MySQL directly as back-end storage and needs to compile itself. At present, we have removed the MySQL client driver for license reasons, but in order to make users use it a little more conveniently, I think it is necessary to keep such a MySQL schema here.

Reply:
fine to me, but we should write somewhere the instructions about how to use MySQL.
If we are maintaining the compatibility it should be clear for the community